apps/src/code-studio/components/libraries/LibraryListItem.jsx

```diff
@@ -89,7 +85,7 @@ export default class LibraryListItem extends React.Component {
           </button>
         )}
         <span style={styles.description}>
-          {this.displayDescription(library.description)}
+          {library.description}
           <br />
           {library.studentName && (
             <span style={styles.author}>Author: {library.studentName}</span>
```

Comment:
Nit, not related to this PR: localization of this string?

Reply:
Good call. I made a task go do a final pass of library UI elements and localize them.
https://codedotorg.atlassian.net/browse/STAR-788